src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -13368,6 +13368,53 @@ public sealed partial class TypeForwardedToAttribute : System.Attribute
         public TypeForwardedToAttribute(System.Type destination) { }
         public System.Type Destination { get { throw null; } }
     }
+    [System.CLSCompliantAttribute(false)]
+    public static partial class Unsafe
```

Comment:
Hmm, this does not look right. Why is it here?

Reply:
Because member of the class are non-compliance. Did you want to decorate each of them individually?

Reply:
We typically annotate individual members rather than splitting them apart into separate partials.

Reply:
Changed to per method.